notebooks/duckdb-writer.ipynb

```diff
@@ -47,6 +55,16 @@
     "    return doc"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "aryn_api_key = os.environ.get(\"ARYN_API_KEY\") ## REPLACE WITH YOUR-ARYN-API-KEY\n",
```

Comment:
looks good to me, just one thing. Jon wants to pass api key as a string in notebooks, can you check with him also ?